src/WorkflowExecuter/Services/WorkflowExecuterService.cs

```diff
@@ -319,25 +318,40 @@ private async Task<bool> UpdateWorkflowInstanceStatus(WorkflowInstance workflowI
 
         private async Task HandleDicomExport(WorkflowRevision workflow, WorkflowInstance workflowInstance, TaskExecution task, string correlationId)
```

Comment:
function name is missing Async
should be HandleDicomExportAsync

Reply:
Changed